src/expression/types/projection_definition.ts

```diff
@@ -7,10 +7,11 @@ export class ProjectionDefinition {
         this.from = from;
         this.to = to;
         this.transition = transition;
+        
     }
 
     static interpolate(from: string, to: string, t: number) {
-        return new ProjectionDefinition(from, to, t);
+        return new ProjectionDefinition(from, to, t).toJSON();
```

Comment:
I think interpolate should return the same type of object and not a string...

Reply:
Okay,

Just to clarify, this does return an array, not a string.